src/Microsoft.Diagnostics.Monitoring.WebApi/Metrics/PrometheusDataModel.cs

```diff
@@ -58,6 +74,45 @@ public static string GetPrometheusNormalizedValue(string unit, double value)
             return value.ToString(CultureInfo.InvariantCulture);
         }
 
+        private static void NormalizeLabelValue(StringBuilder builder, string value)
+        {
+            for (int i = 0; i < value.Length; i++)
+            {
+                if (value[i] == SlashChar)
+                {
+                    builder.Append(SlashChar);
+                    if (i < value.Length - 1 && value[i + 1] == SlashChar)
+                    {
+                        builder.Append(SlashChar);
+                        ++i; // Skip over the second (already escaped) slash
+                    }
+                    else if (i < value.Length - 1 && value[i + 1] == 'n')
+                    {
+                        builder.Append('n');
+                        ++i; // Skip over the 'n' in the newline
+                    }
+                    else if (i < value.Length - 1 && value[i + 1] == QuotationChar)
+                    {
+                        builder.Append(QuotationChar);
+                        ++i; // Skip over the '"'
+                    }
+                    else
+                    {
+                        builder.Append(SlashChar);
+                    }
```

Comment:
This doesn't seem quite right. I think it should just escape a backslash as two backslashes, similar to how you are handling the double quote. Separately, you need to handle when `value[i] == '\n'`, and write that out as a literal backslash and a literal `n`.